app/src/main/res/values/strings.xml

```diff
@@ -176,26 +176,26 @@
-    <string name="exception_table">Table dictionary (.dict or .txt) not found</string>
-    <string name="table_im">Table Input Methods</string>
-    <string name="manage_table_im">Manage Table Input Methods</string>
+    <string name="exception_table_im">Configuration file of table-based input method (.conf or .conf.in) not found</string>
+    <string name="exception_table">Dictionary of table-based input method (.dict or .txt) not found</string>
+    <string name="table_im">Table-based Input Methods</string>
+    <string name="manage_table_im">Manage Table-based Input Methods</string>
     <string name="exception_table_im_filename">The extension of \"%1$s\" does not indicate a table input method package (.zip)</string>
-    <string name="table_already_exists">Table input method configuration file \"%1$s\" already exists</string>
-    <string name="table_file_does_not_exist_title">Table unavailable</string>
-    <string name="table_file_does_not_exist_message">Table dictionary \"%1$s\" does not exist. Please check the dict file or re-import.</string>
-    <string name="import_table">Import Table</string>
+    <string name="table_already_exists">Configuration file of table-based input methods \"%1$s\" already exists</string>
+    <string name="table_file_does_not_exist_title">Character mapping table unavailable</string>
+    <string name="table_file_does_not_exist_message">Dictionary of table-based input methods \"%1$s\" does not exist. Please check the dict file or re-import.</string>
+    <string name="import_table">Import character mapping table</string>
     <string name="table_from_zip">From zip…</string>
     <string name="table_select_files">From separate files…</string>
     <string name="table_file_placeholder">Select file…</string>
     <string name="table_conf_select_title">Input method configuration file (.conf or .conf.in)</string>
-    <string name="table_dict_select_title">Table dictionary (.dict or .txt)</string>
-    <string name="exception_table_conf_filename">The extension of \"%1$s\" does not indicate a table input method configuration file (.conf or .conf.in)</string>
-    <string name="exception_table_dict_filename">The extension of \"%1$s\" does not indicate a table dictionary (.dict or .txt)</string>
-    <string name="exception_table_import_both_files">Please select both of above files before importing</string>
+    <string name="table_dict_select_title">Dictionary of table-based input methods (.dict or .txt)</string>
+    <string name="exception_table_conf_filename">The extension of \"%1$s\" does not indicate a configuration file of table-based input methods (.conf or .conf.in)</string>
+    <string name="exception_table_dict_filename">The extension of \"%1$s\" does not indicate a dictionary of table-based input method (.dict or .txt)</string>
+    <string name="exception_table_import_both_files">Please select at least two files to import</string>
```

Comment:
This string is used for an error message when the user triggers import somehow without selecting both of the .conf and .txt/dict files. Normally, the import button should be disabled when either of those files was not selected.

https://github.com/fcitx5-android/fcitx5-android/blob/10ff08f79b7cefbb2cb959e57fe49f89b2edb8b3/app/src/main/java/org/fcitx/fcitx5/android/ui/main/settings/TableInputMethodFragment.kt#L276-L279

Since both of those files are required, and there is not an option to select 1 or 3 files for importting, I doubt whether it's suitable to use "at least two files" here.

Reply:
The Simplified Chinese string of `exception_table_import_both_files` is `请先选择以上两个文件再导入`, which is also questionable.

Reply:
I didn't see why it's questionable because both of those files are required, and only one of each type can be selected. How would you explain such constrains in Chinese?

Reply:
I tried importing mapping table by myself, in each type only one file can be selected, therefore it is unambiguous.